tests/platform_tests/test_kdump.py

```diff
@@ -27,12 +27,16 @@ class TestKernelPanic:
     def wait_lc_healthy_if_sup(self, duthost, duthosts, localhost, conn_graph_facts, xcvr_skip_list):
         # For sup, we also need to ensure linecards are back and healthy for following tests
         is_sup = duthost.get_facts().get("modular_chassis") and duthost.is_supervisor_node()
+        if 'chassis-packet' in duthost.facts.get('switch_type'):
```

Comment:
Does 'chassis-packet' mean ethernet packet-based chassis rather than cell-based chassis?
If so, I suppose the hearbeat-loss behavior is platform related?
So we better to use hwsku to decide the logic, otherwise in the future if there's another packet-based chassis but not Cisco platform, the logic would be wrong here?

Reply:
Another question is, do you expect the test_kdump to fail with existing code?
In our newest nightly test run, it's passing

Reply:
Hi @yejianquan - I believe Cisco chassis is the only chassis that is chassis-packet rest all are voq based. Hence the check used. We have other places too where this check is used for our chassis. 

Regarding the behavior, when kernel panic happens, RP goes for reboot ungracefully - hence LC reboot cause is headless RP/heartbeat loss. THe reboot on LC is graceful but the cause is due to RP getting rebooted ungracefully

Reply:
Made the change that you requested @yejianquan